<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_op">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>55</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(     )</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_d1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>digito 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_d2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>56</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>digito 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Nuevo Juego</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ingtresa">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingresa tu Resultado:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_resultado">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Comprobar</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>97</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>SUMA</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>97</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>RESTA</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>MULTIPLICACION</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>97</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>DIVISION</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
